bagel/utilities/pheno_utils.py

```diff
@@ -204,18 +204,21 @@ def transform_age(value: str, value_format: str) -> float:
             return float(value.strip("+"))
         if value_format == AGE_FORMATS["iso8601"]:
             if not value.startswith("P"):
-                value = "P" + value
-            duration = isodate.parse_duration(value)
+                pvalue = "P" + value
+            else:
+                pvalue = value
+            duration = isodate.parse_duration(pvalue)
             return float(duration.years + duration.months / 12)
         if value_format == AGE_FORMATS["range"]:
-            return sum(map(float, value.split("-"))) / 2
+            a_min, a_max = value.split("-")
+            return sum(map(float, [a_min, a_max])) / 2
```

Comment:
So I guess this raises a `ValueError` if one of the two values ends up being `None`, yes?

Reply:
indeed!